test/basic.mixed.expect.css

```diff
@@ -1,5 +1,5 @@
-:root {
+html {
```

Comment:
I'm not really sure how this test was passing before, because it uses `postcss-unroot`. It started failing once I updated `postcss-tape`.